deployment/ansible/roles/raster-foundry.jenkins/templates/jenkins.conf.j2

```diff
@@ -0,0 +1,17 @@
+server {
+    listen 80;
+    server_name {{ jenkins_server_name }};
+    proxy_redirect off;
```

Comment:
Fixed in [`564b758`](https://github.com/azavea/raster-foundry/pull/419/commits/564b758563a02b4b95630378f8995edeaa9c49dc)